examples/layers/preprocessing/cut_out_demo.py

```diff
@@ -0,0 +1,47 @@
+"""cut_out_demo.py shows how to use the CutOut preprocessing layer.
+
+Operates on the oxford_flowers102 dataset.  In this script the flowers 
+are loaded, then are passed through the preprocessing layers.  
+Finally, they are shown using matplotlib.
+"""
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from keras_cv.layers.preprocessing import erasing
```

Comment:
Recently I pushed a commit importing Preprocessing layers to the __init__.py file for `keras_cv/layers/preprocessing/__init__.py`.  Let's also import these layers there, and then we can do:

```
from keras_cv.layers import preprocessing
preprocessing.CutOut
```